src/main.rs

```diff
@@ -38,7 +60,10 @@ main!(|args: Cli| {
         project_dir.display()
     );
 
-    let _template = GitRepository::clone(&args.git, &project_dir)
+    let _template = RepoBuilder::new()
+        .bare(false)
+        .with_checkout(CheckoutBuilder::new())
+        .clone(&args.git, &project_dir)
         .with_context(|_e| format!("Couldn't clone `{}`", &args.git))?;
 
     fs::remove_dir_all(&project_dir.join(".git")).context("Error cleaning up cloned template")?;
```

Comment:
The error here is a common issue with Windows and Git. The TL;DR is that git creates read-only files, which windows then refuses to remove; in essence, on NIX, you can delete a read-only file, but not on Windows.

An easy fix is to use https://crates.io/crates/remove_dir_all

Some sample code: https://github.com/steveklabnik/doxidize/blob/master/src/ops/clean.rs

Reply:
I can confirm this fails locally. Trying out that crate:

```diff
diff --git a/src/main.rs b/src/main.rs
index cc8694a..61e87f8 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -7,6 +7,7 @@ extern crate indicatif;
 extern crate liquid;
 extern crate regex;
 extern crate walkdir;
+extern crate remove_dir_all;

 use dialoguer::Input;
 use git2::{
@@ -66,7 +67,7 @@ main!(|args: Cli| {
         .clone(&args.git, &project_dir)
         .with_context(|_e| format!("Couldn't clone `{}`", &args.git))?;

-    fs::remove_dir_all(&project_dir.join(".git")).context("Error cleaning up cloned template")?;
+    remove_dir_all::remove_dir_all(&project_dir.join(".git")).context("Error cleaning up cloned template")?;

```

(with the appropriate add in cargo.toml) errors for a different reason:

```
failures:

---- basics::simple_cargo_thing stdout ----
thread 'basics::simple_cargo_thing' panicked at 'Unexpected failure.
code-1
stderr=``````
command=`"git" "commit" "--message" "initial commit"`
code=1
stdout=```On branch master
nothing to commit, working tree clean
```
stderr=``````
', C:\Users\steve\.cargo\registry\src\github.com-1ecc6299db9ec823\assert_cmd-0.4.0\src\assert.rs:140:13
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```